pjsip/include/pjsua-lib/pjsua_internal.h

```diff
@@ -79,12 +79,14 @@ struct pjsua_call_media
                                          bit 0/LSB : sequence flag
                                          bit 1     : timestamp flag         */
 
-    pjmedia_type                prev_type;     /**< Previous media type     */
-    pjmedia_stream_info         prev_aud_si;   /**< Prev audio stream info  */
-    pjmedia_vid_stream_info     prev_vid_si;   /**< Prev video stream info  */
-    pj_bool_t                   prev_srtp_use; /**< Prev SRTP use           */
-    pjmedia_srtp_info           prev_srtp_info;/**< Prev SRTP transport info*/
-    pj_bool_t                   prev_ice_use;  /**< Prev ICE use            */
+    pjmedia_type         prev_type;            /**< Previous media type     */
+    pj_sockaddr          prev_aud_local_addr;  /**< Prev aud local address  */
+    pj_sockaddr          prev_aud_rem_addr;    /**< Prev aud remote address */
+    pj_sockaddr          prev_vid_local_addr;  /**< Prev vid local address  */
+    pj_sockaddr          prev_vid_rem_addr;    /**< Prev vid remote address */
```

Comment:
Can `aud` & `vid` in local & remote addresses be merged or perhaps use union for a little bit more optimization?

Reply:
I'm not sure how to merge them since they're used at the same time?
https://github.com/pjsip/pjproject/pull/2846/files#diff-29def4cfd6b2e0bb7bc454130942b8d60a375d2fa5ac8aece3bef6639c569e83R3346

Reply:
`pjsua_call_media` is supposed to be one media type (i.e: either audio or video) at a time and cannot be both?
So, the pointed code is supposed to be `if` + `else (if)` instead of two `if`s perhaps?